packs/aws/requirements.txt

```diff
@@ -1 +1,2 @@
 boto
+boto3
```

Comment:
That's fine, lets just add a comment to the requirements.txt file which says that boto3 is used by SQS sensor.